### PR TITLE
TDS5 NCSS fixes

### PR DESCRIPTION
**Pull request by dopplershift**
This fixes some errors and a bunch of warnings in parsing the `dataset.xml` on a TDS 5.0 grib collection.

### CODE REVIEW

File: siphon/ncss_dataset.py [hunk 1]
```diff
@@ -76,16 +77,14 @@ def handle_typed_values(val, type_name, value_type):
         """
         if value_type in ['byte', 'short', 'int', 'long']:
             try:
-                val = val.split()
-                val = list(map(int, val))
+                val = [int(v) for v in re.split('[ ,]', val) if v]
             except ValueError:
-                log.warning('Cannot convert %s to int. Keeping type as str.', val)
```
Comment (jrleeman) [inline siphon/ncss_dataset.py]: Can't do `.format`?

Comment (dopplershift) [reply siphon/ncss_dataset.py]: Nope, logging is old-style only.

**Comment by dopplershift** [comment]
Not exactly...I replaced `['a']` in the output message with `"a"`. The former was confusing as I was trying to figure out what was going on, so I switched to something that seemed clearer.

**Comment by dopplershift** [comment]
Looks like the AppVeyor failure was transient. 🎉

**Pull request merged**